content/release.rst

```diff
@@ -86,23 +86,26 @@ Some functions of the **release manager** are:
   to make sure no regressions were introduced on the tests executed post-merge.
 * Creating high priority tickets in case the tests are failing on master.
 * Coordinating story tickets for the milestone.
-
+* After sending the release branch to RQM, check that the Downloads page is updated.
+* After the Downloads page is updated, ensure that the release branch is merged
+  with the master branch.
+* Sends the newsletter to the relevant list/s.
 
 Release Manager should look into obtaining access to the following:
 
-* Write access to sftpplus.com production (from infrastructure team) to
+* Write access to production website (from infrastructure team) to
   the news release to the website.
 * Ability to stage a release branch to staging server then to production
   server.
 * Access to Mailchimp to send the release newsletter.
 * Access to the Support helpdesk or emails to know which customers should be
   contacted directly if the release is awaiting upon them.
 
-
 When the release is out, the Release Manager organizes the team release
 meeting (times and dates), initiates the call and holds the meeting including
 a distributed agenda.
-A template of the release meeting, including past notes is `located here <https://drive.google.com/drive/folders/0B91muor_IWXBaHp1eExRbGcyZ28?usp=sharing>`_
+Release meeting notes are `located here <https://drive.google.com/drive/u/3/folders/0BwQo7116Iy2tZ2M2bDhadFV4R0E>`_
```

Comment:
If we have a template, it should be public. Maybe on a dedicated page inside this site.

Reply:
We don't have templates just release meeting notes.  I thought we did.

Reply:
For templates, we should store the templates only in the styleguide... so that we don't need to duplicate them